build.sh

```diff
@@ -10,26 +10,52 @@ export BUILD_ROLLING="${1:-no}"
 
 DOCKER=docker
 
+# The following variables can be overridden using env-variables
+
+# Set tag namespace (i.e. ghcr.io/<org> for github package registry)
+TAG_NS="${TAG_NS:-maxking}"
+# Set default platforms to build
+PLATFORM="${PLATFORM:-linux/arm64/v8,linux/amd64}"
+# Platform to load into docker after build
+# Can only load one platform, should match host
+PLATFORM_TO_LOAD="linux/amd64"
```

Comment:
Should we figure out the current platform instead of hard coding it? Or, at least provide a way to override it using env var so someone can run builds on non-x86 machines too?

Also, let's rename the vars as `BUILD_PLATFORM` and `CURRENT_PLATFORM` instead? I don't think `PLATFORM_TO_LAOD` not matching the current platform would really work.